reduct-nepal/seep
# Legend added in the diff view page

**Task Completed:

[ ] Added Legend in view diff page

**To test:

- Log into SEEP as superuser
- go to admin section
- goto student records
- click on view button to goto transcription page 

**Expectations:

- Superuser should see the Legend along with the student info button


**Intentions:

To let evaluator know what different color in the transcription page suggest
